shopinvader_product_stock/models/stock_move.py

```diff
@@ -57,7 +54,6 @@ def _action_confirm(self, merge=True, merge_into=False):
         result._jobify_product_stock_update()
         return result
 
-    @api.multi
     def _action_done(self):
```

Comment:
cancel_backorder key is missing in _action_done method
```
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 624, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/opt/odoo/custom/src/odoo/odoo/tools/pycompat.py", line 14, in reraise
    raise value
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 669, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 350, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 915, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/auto/addons/web/controllers/main.py", line 1326, in call_button
    action = self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/auto/addons/web/controllers/main.py", line 1314, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 387, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 374, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/auto/addons/stock/models/stock_picking.py", line 910, in button_validate
    self.action_done()
  File "/opt/odoo/auto/addons/shopinvader_delivery_carrier/models/stock_picking.py", line 61, in action_done
    result = super(StockPicking, self).action_done()
  File "/opt/odoo/auto/addons/purchase_request/models/stock_picking.py", line 36, in action_done
    super(StockPicking, self).action_done()
  File "/opt/odoo/auto/addons/purchase_reception_notify/models/stock_picking.py", line 30, in action_done
    super(StockPicking, self).action_done()
  File "/opt/odoo/auto/addons/quality_control/models/stock_picking.py", line 59, in action_done
    return super(StockPicking, self).action_done()
  File "/opt/odoo/auto/addons/stock/models/stock_picking.py", line 727, in action_done
    todo_moves._action_done(cancel_backorder=self.env.context.get('cancel_backorder'))
TypeError: _action_done() got an unexpected keyword argument 'cancel_backorder'
```

Reply:
@HviorForgeFlow sorry, I'm working on something else and had not time to test it yet. I pushed a fix.